pyDrop/__init__.py

```diff
@@ -5,3 +5,5 @@
 
 description here
 """
+from .clustering import *
+from .exceptions import *
```

Comment:
suggestion: pep8 is a bit circumspect about relative vs absolute imports, but does come down preferring absolute imports.